app/graphql/types/customers/object.rb

```diff
@@ -36,7 +36,14 @@ class Object < Types::BaseObject
       field :billing_configuration, Types::Customers::BillingConfiguration, null: true
 
       field :provider_customer, Types::PaymentProviderCustomers::Provider, null: true
-      field :subscriptions, [Types::Subscriptions::Object]
+      field :subscriptions, [Types::Subscriptions::Object], resolver: Resolvers::Customers::SubscriptionsResolver
+
+      field :invoices, [Types::Invoices::Object]
+
+      field :applied_coupons, [Types::AppliedCoupons::Object], null: true
+      field :applied_add_ons, [Types::AppliedAddOns::Object], null: true
```

Comment:
GraphQL/OrderedFields: Fields should be sorted in an alphabetical order within their section. Field `applied_add_ons` should appear before `applied_coupons`.